synapseclient/models/schema_organization.py

```diff
@@ -0,0 +1,1314 @@
+"""
+This file contains the SchemaOrganization and JSONSchema classes.
+These are used to manage Organization and JSON Schema entities in Synapse.
+"""
+
+import re
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, Any, Optional, Protocol
+
+from synapseclient.api import (
+    create_organization,
+    delete_json_schema,
+    delete_organization,
+    get_json_schema_body,
+    get_organization,
+    get_organization_acl,
+    list_json_schema_versions,
+    list_json_schemas,
+    list_organizations_sync,
+    update_organization_acl,
+)
+from synapseclient.core.async_utils import async_to_sync
+from synapseclient.core.constants.concrete_types import CREATE_SCHEMA_REQUEST
+from synapseclient.models.mixins.asynchronous_job import AsynchronousCommunicator
+from synapseclient.models.mixins.json_schema import JSONSchemaVersionInfo
+
+if TYPE_CHECKING:
+    from synapseclient import Synapse
+
+SYNAPSE_SCHEMA_URL = (
+    "https://repo-prod.prod.sagebase.org/repo/v1/schema/type/registered/"
```

Comment:
We cannot assume that the URL would be production. This would need to come from the configuration file that is being used. You would want to use the `repoEndpoint` on the Synapse class instance in order to get the base of the URL that you are using.